### PR TITLE
Envar filter and remove

### PR DESCRIPTION
**Pull request by mgonzalezperna**

### CODE REVIEW

File: resin_release_tool/cli.py [hunk 1]
```diff
@@ -118,5 +118,50 @@ def releases(releaser, count):
...
+@pass_releaser
+@click.pass_context
+def filter_and_remove_env_var(ctx, releaser, envar_model, \
+                              envar_name, envar_value, exclusive_condition: bool):
+    """Filter devices by enviroment variables and removes them."""
+
+    try:
+        devices = releaser.get_devices_filtered_by_condition(
+            envar_model, envar_name, envar_value, exclusive_condition)
+    except Exception as exception:
```
Comment (Roger) [inline resin_release_tool/cli.py]: this exception is too broad

File: resin_release_tool/releaser.py [hunk 2]
```diff
@@ -21,6 +21,237 @@ def get_canaries(self):
...
+        else:
+            raise ValueError(
+                f'The environment {environment_model} model selected is invalid')
+        return results
+
+    def _remove_envar_of_app(self, envar):
+        try:
+            self.models.environment_variables.application.remove(envar['id'])
+            results = 'Success!'
+        except:
```
Comment (Roger) [inline resin_release_tool/releaser.py]: this exception is too broad

File: resin_release_tool/releaser.py [hunk 3]
```diff
@@ -21,6 +21,237 @@ def get_canaries(self):
...
+            results = 'Failed'
+        return results
+
+    def _remove_envar_of_serv(self, envar):
+        try:
+            self.models.environment_variables.\
+                service_environment_variable.remove(envar['id'])
+            results = 'Success!'
+        except:
```
Comment (Roger) [inline resin_release_tool/releaser.py]: this exception is too broad

File: resin_release_tool/releaser.py [hunk 4]
```diff
@@ -21,6 +21,237 @@ def get_canaries(self):
...
+
+    def _remove_envar_of_devserv(self, envar_name, devices):
+        results = {'done':0, 'failed':0}
+        for device in devices.values():
+            envar_id = self._get_envars_info_devserv(device)[envar_name]['id']
+            try:
+                self.models.environment_variables.\
+                    device_service_environment_variable.remove(envar_id)
+                results['done'] += 1
+            except:
```
Comment (Roger) [inline resin_release_tool/releaser.py]: this exception is too broad

Comment (mgonzalezperna) [reply resin_release_tool/releaser.py]: Fixed

File: resin_release_tool/releaser.py (hunk 3 above)
Comment (mgonzalezperna) [reply resin_release_tool/releaser.py]: Fixed.

File: resin_release_tool/releaser.py (hunk 2 above)
Comment (mgonzalezperna) [reply resin_release_tool/releaser.py]: Fixed.

File: resin_release_tool/cli.py (hunk 1 above)
Comment (mgonzalezperna) [reply resin_release_tool/cli.py]: Fixed.

File: resin_release_tool/cli.py [hunk 5]
```diff
@@ -118,5 +118,50 @@ def releases(releaser, count):
...
+@cli.command()
+@click.argument('envar_model')
+@click.argument('envar_name')
+@click.argument('envar_value')
+@click.option('--exclusive', is_flag=True, default=False)
+@pass_releaser
+@click.pass_context
+def filter_and_remove_env_var(ctx, releaser, envar_model, \
+                              envar_name, envar_value, exclusive):
+    """Filter devices by enviroment variables and removes them."""
```
Comment (Roger) [inline resin_release_tool/cli.py]: Can you add more documentation here, like that each variable means and help to exclusive?

Comment (mgonzalezperna) [reply resin_release_tool/cli.py]: Sure! On it

**Pull request closed without merging**